consensus/interface.go

```diff
@@ -165,11 +165,12 @@ type PeerBlacklistHandler interface {
 	IsInterfaceNil() bool
 }
 
-// SignatureHandler defines the behaviour of a component that handles signatures in consensus
-type SignatureHandler interface {
+// SigningHandler defines the behaviour of a component that handles multipart and single signatures used in consensus operations
```

Comment:
can you rename multipart to multi signature?

Reply:
replaced